gooddata-sdk/gooddata_sdk/catalog/base.py

```diff
@@ -65,3 +70,7 @@ def client_class() -> Any:
     def to_api(self) -> Any:
         dictionary = self._get_snake_dict()
         return self.client_class().from_dict(dictionary, camel_case=False)
+
+    @staticmethod
+    def _custom_converter() -> Optional[cattr.GenConverter]:
```

Comment:
Make custom converted result singleton. Current implementation creates as new converter for each `from_api` call but it is a static object.